spec/requests/api/arbitration_profile_spec.rb

```diff
@@ -74,6 +149,33 @@
       expect(default.reload.cloud_subnet_id).to eq(subnet.id)
     end
 
+    it 'supports edit with a provider id or href specified' do
```

Comment:
Did you mean to test the href case here too? (and in the tests below it). I'd think it would be OK not to (if your other tests give you sufficient confidence that hrefs can be used) but I would just update the test description to not mention which you are using as a means of forming a valid request